### ISSUE TITLE
ODRL Community Vocabulary

### ISSUE THREAD

**Issue by riannella** [w3c/odrl#43]
- A community-supplied set of general terms
  - Have a few items to add to the vocab to kick-start the draft (see past discussions)
  - What will be the namespace?
  - Use the same ttl or different files ?
  - Process to follow ?
  - Who will be the Editors? 
  - Auto-generate from a ttl file ?

**Comment by bact** [comment]
On the namespace https://lists.w3.org/Archives/Public/public-odrl/2024Feb/0004.html

**Comment by vroddon** [comment]
I add @besteves4, for she volunteers to create a spreadsheet with all the terms that have been discussed.

**Comment by besteves4** [comment]
Hi, I created a spreadsheet [here](https://docs.google.com/spreadsheets/d/1wSzrpyETSC0rV4wVcVgBP0OO9C-n9J384mtTIrN11hk/edit?usp=sharing) to manage the community vocabulary terms.

**Comment by riannella** [comment]
Yes, it is possible. The W3C recommendations terms are standardised. The URI is ours to extend. It makes it less confusing to use the same URI.
